<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1026</width>
    <height>740</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="fieldWindow">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>381</width>
      <height>421</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="infoWindow">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>60</y>
      <width>271</width>
      <height>421</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="logsWindow">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>60</y>
      <width>271</width>
      <height>421</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="addNeutralButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>500</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Neutral</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelNeutralType">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>540</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Neutral type</string>
    </property>
   </widget>
   <widget class="QFontComboBox" name="typesOfNeutral">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>540</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="neutralPosX">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>600</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelNeutralPosition">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>570</y>
      <width>59</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Position</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="neutralPosY">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>600</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="addUnitButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>500</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Unit</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelUnitType">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>540</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Unit type</string>
    </property>
   </widget>
   <widget class="QFontComboBox" name="typesOfUnits">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>540</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelUnitPosition">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>570</y>
      <width>59</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Position</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="unitPosX">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>600</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="unitPosY">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>600</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelUnitBase">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>630</y>
      <width>59</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Unit base</string>
    </property>
   </widget>
   <widget class="QFontComboBox" name="fontComboBox_3">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>630</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="addBaseButton">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>500</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Add base</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelBasePosition">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>540</y>
      <width>59</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Position</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="basePosX">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>570</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="basePosY">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>570</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="moveUnitButton">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>500</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Move Unit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="attackButton">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>500</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Attack</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelStep">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>540</y>
      <width>59</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Step</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="stepPosX">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>570</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="stepPosY">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>570</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="unitInfoButton">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>610</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>unit info</string>
    </property>
   </widget>
   <widget class="QPushButton" name="neutralInfoButton">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>640</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>neutral  info</string>
    </property>
   </widget>
   <widget class="QPushButton" name="landscapeInfoButton">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>670</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>landscape info</string>
    </property>
   </widget>
   <widget class="QPushButton" name="baseInfoButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>660</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>base info</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelPositionInfoItem">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>610</y>
      <width>59</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Position</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="infoItemPosX">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>640</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="infoItemPosY">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>640</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="gameInfoButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>game info</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelField">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>59</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Field</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelInfo">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>40</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Information about item</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelLogs">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>40</y>
      <width>59</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Logs</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>610</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>max count of units</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>650</y>
      <width>59</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>health</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="basePosX_2">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>630</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="basePosX_3">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>670</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
